draft-ietf-scitt-architecture.md

```diff
@@ -639,31 +639,24 @@ Although this weakens their guarantees against key revocation, or against a corr
 
 This section details the interoperability requirements for implementers of Signed Statements issuance and validation libraries, and of Transparency Services.
 
-##  Envelope and Signed Statement Format
+##  Issuing Signed Statements
```

Comment:
@hannestschofenig : As the Main heading for the section is `Signed Statement Issuance, Registration, and Verification`

I think, it makes sense to introduce this section as `Signed Statement Envelope` The reason been (a) We are not discussing Issuance per say here

(b) We do not have a fixed signed statement format, but we just have an Envelope which contains a `Payload`